docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
+.Procedure
+
+. In the navigation pane, click *Projects*.
+. On the *Projects* page, click *Create project*.
```

Comment:
Take out "On the Projects page". It's obvious that they're on that page because they just clicked Projects.